contractsSol5/mock/GenerousKyberNetwork2.sol

```diff
@@ -0,0 +1,96 @@
+pragma solidity 0.5.11;
+
+
+import "../KyberNetwork.sol";
+
+
+////////////////////////////////////////////////////////////////////////////////////////////////////////
+/// @title GenerousKyberNetwork2 transfer the fixed dest amount to destAddress and returns this amount to proxy
+// This would allow us to check the condition of maxDestAmount
+contract GenerousKyberNetwork2 is KyberNetwork {
+
+    constructor(address _admin) public KyberNetwork(_admin) { }
+
+    event GenerousTrade(int which, int more, IERC20 token);
+    /* solhint-disable function-max-lines */
+    /// @notice use token address ETH_TOKEN_ADDRESS for ether
+    /// @dev trade api for kyber network.
+    /// @param tData.input structure of trade inputs
+    function trade(TradeData memory tData, bytes memory hint)
+        internal
+        nonReentrant
+        returns(uint destAmount)
+    {
+        tData.networkFeeBps = getAndUpdateNetworkFee();
+
+        // destAmount = printGas("start_tr", 0, Module.NETWORK);
```

Comment:
please remove all the print Gas lines

Reply:
updated in https://github.com/KyberNetwork/smart-contracts/pull/726/commits/e4d067f71b9400d43a678940a586bbb43991f02a